config/locales/en.yml

```diff
@@ -102,6 +102,8 @@ en:
           expired: "The access token expired"
           unknown: "The access token is invalid"
 
+        my_message: 'This is my authorization custom message'
```

Comment:
Agree I've already thought about it. What about adding a new file in the same folder `en_custom_errors.yml` with:
```yml
my_message: 'This is my authorization custom message'
```